app/src/main/res/authors.txt

```diff
@@ -56,10 +56,15 @@ fire_hydrant_pond.jpg          CC-BY-SA 3.0      KeepOnTruckin https://commons.w
 fire_hydrant_underground.jpg   CC-BY 3.0         Panek https://commons.wikimedia.org/wiki/File:Hydrant_MPWiK.jpg
 fire_hydrant_wall.jpg          CC-BY-SA 3.0      Ingo Fick https://commons.wikimedia.org/wiki/File:Guentherscheid_Tunnel_Rescue4.jpg
 
-fire_hydrant_position_green... CC-BY-SA 4.0     https://upload.wikimedia.org/wikipedia/commons/2/2d/Fire_hydrant_in_grass.jpg
-fire_hydrant_position_lane...  CC-BY-SA 4.0     https://upload.wikimedia.org/wikipedia/commons/3/32/Fire_hydrant_on_sidewalk.jpg
-fire_hydrant_position_sidew... CC-BY-SA 4.0     modified from https://upload.wikimedia.org/wikipedia/commons/d/d0/Fire_hydrant_on_road_side.jpg
-fire_hydrant_position_parki... CC-BY 2.0        modified from https://upload.wikimedia.org/wikipedia/commons/3/33/Fire-fighting-facility_node-4279866277.jpg
+fire_hydrant_position_undergr… CC-BY-SA 4.0      Thefeiter https://commons.wikimedia.org/wiki/File:Fire_hydrant_in_grass.jpg
+fire_hydrant_position_undergr… CC-BY-SA 4.0      modified from Thefeiter https://commons.wikimedia.org/wiki/File:Fire_hydrant_on_road_side.jpg
```

Comment:
modification are limited to cropping, right? Maybe drop "modified from" or modify to "cropped from"/"trivially modified" to avoid another problem of "author of modification is not attributed on file which license requires attribution"

Reply:
For the new pillar hydrant images, I cropped and a tweaked brightness/contrast a bit. That seems to be the case for the underground hydrant images, as well.

In this file and `res/graphics/authors.txt`, modification is never described more precisely. Do we need this at all? Note that modification inside this repo is retraceable via the git history.

Reply:
https://commons.wikimedia.org/wiki/File:Fire_hydrant_in_grass.jpg#Licensing - human readable summary has

> You must give appropriate credit, provide a link to the license, and indicate if changes were made

so just indicating changes is enough

but there is also

> share alike – If you remix, transform, or build upon the material, you must distribute your contributions under the same or compatible license as the original.
> attribution – You must give appropriate credit

so I think it would be useful to either make clear that changes were trivial, below https://commons.wikimedia.org/wiki/Commons:Threshold_of_originality or mention who did them to satisfy attribution requirement

(warning: I am not a lawyer)

> modification is never described more precisely. Do we need this at all? 

I may be wrong that it is needed, it may be also true that modifications need to be mentioned also elsewhere

Reply:
Seems like kerfuffle over this particular image could be avoided by `s/modified/cropped/`, which would take less effort than discussing it at all here :P

Reply:
https://github.com/streetcomplete/StreetComplete/pull/3695#discussion_r796453537

Reply:
@FloEdelmann If you think that I am panicking and nitpicking: feel free to merge it and I will do something with it later after merge.

Reply:
Since all the other ones also lack more precise descriptions, and this PR can be retraced via the git history, I think it's fine to merge now.